### ISSUE TITLE
[PNI] Product page - update Blur and Worst case blocks to rich text

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#7449]
- [ ] Update existing "Blur" and "Worst case" blocks to rich text to enable the ability to have plain text, bold, italics, links

**Comment by richbrennan** [comment]
I wasn't able to recreate the "pickling" errors locally to investigate. I tried converting the Blurb field to rich text and adding some RT formatted content but no errors were thrown. Is it possible to proved the steps needed to reproduce please?

**Comment by Pomax** [comment]
Absolutely. @danielfmiranda could I ask you to put the code swap you wrote into this issue? Hopefully this was just a matter of using the wrong class =)

**Comment by danielfmiranda** [comment]
Hi @Pomax, of course! One thing I would like to note however is that I am experiencing the error in[ this ticket ](https://github.com/mozilla/foundation.mozilla.org/pull/7477/) which has to do with adding the "tips to protect yourself streamfield", vs updating blurb and worst case like requested in this ticket. 

If you would like to take a look at the changes I made in order to use a rich text block streamfield, the file changes can be found [here](https://github.com/mozilla/foundation.mozilla.org/pull/7477/files) under the file **products.py**, and you can see what I am doing to render all of them on the front end in **product_page.html**

Please let me know if this helps, or if you need anything else. Thanks!

**Comment by richbrennan** [comment]
Hi @danielfmiranda 

Thanks for the links. Is this still an issue on the newer PR? https://github.com/mozilla/foundation.mozilla.org/pull/7576

If so I'm happy to check out the branch and take a look.

Cheers,

Rich.

**Comment by danielfmiranda** [comment]
Hi @richbrennan! Thanks for checking in, this is no longer an issue on the PR you linked, and it has been merged in to the codebase. I believe that we are now good to go in terms of working on this ticket

**Comment by danielfmiranda** [comment]
Closing as this is merged in. Thanks everyone for the review!
